common.mk

```diff
@@ -67,6 +67,12 @@ $(SMEMS_FILE) $(SMEMS_FIR): $(SMEMS_CONF)
 .PHONY: verilog
 verilog: $(sim_vsrcs)
 
+#########################################################################################
+# helper rules to run simulator
+#########################################################################################
+run-binary: $(sim)
+	$(sim) $(PERMISSIVE_ON) $(SIM_FLAGS) $(PERMISSIVE_OFF) $(BINARY) 3>&1 1>&2 2>&3 | spike-dasm > $(sim_out_name).out
```

Comment:
You should pipe to `$(sim_out_name).out`, and then pipe that to `spike-dasm`. I like having the raw output around, it makes it easier to track down instruction bits.

Reply:
For now, I'll leave this so I can move to other stuff. But you should PR it!